### ISSUE TITLE
More helpful message when targeting missing stanzas

### ISSUE THREAD

**Issue by terrorobe** [pgmasters/backrest#74]
When supplying a non-existant stanza the operation should fail in every case with a relevant error message.

At the moment the --stanza directive just seems to interpolate values from the configuration

```
$ /opt/backrest/bin/pg_backrest.pl backup --stanza=NONEXISTANT --type=full
2015-05-19 11:32:51.658 T00  ERROR: [112] backup operation requires option: db-path
```

**Comment by dwsteele** [comment]
Since everything can be supplied on the command line it's a bit tough to figure out if the stanza is "missing".  db-path is one of the few required params in backrest.  What if I added hints?  Like:
```
2015-05-19 11:32:51.658 T00  ERROR: [112] backup operation requires option: db-path
                             HINT: Is the stanza valid?
```

**Comment by terrorobe** [comment]
I'd just hard-fail if you supply a stanza name that is not present in the configuration file if this is feasible with the current config parser.

If not your approach is fine though I think I'd word it along the lines of "Does this stanza exist?"

**Comment by dwsteele** [comment]
Well, anything is possible with time but in general this is not currently supported.  The config parser searches in the conf file for anything it could not resolve on the command line (and before it applies defaults).  It might make multiple passes to resolve dependencies so it's not clear to me where I would add a "stanza is missing" message.

As the parser evolves this might be possible but for now I think I'll go with my approach (though I'm happy with your wording).

**Comment by terrorobe** [comment]
:+1:

**Comment by dwsteele** [comment]
Fixed in 6d2f2d3.
